dolphinscheduler-service/pom.xml

```diff
@@ -82,5 +88,153 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <scope>test</scope>
```

Comment:
We already has these dependent in common module, does this not work?

Reply:
It just use for some UT case. If not add, the UT will fail.